src/dataframe/io.jl

```diff
@@ -561,11 +561,9 @@ function builddf(rows::Integer,
                     continue
                 else
                     msgio = IOBuffer()
-                    @printf(msgio,
-                            "Failed to parse '%s' using type '%s'",
-                            bytestring(p.bytes[left:right]),
-                            o.eltypes[j])
-                    error(bytestring(msgio))
+                    print(msgio, "Failed to parse '", p.bytes[left:right],
+                        "' using type ", o.eltypes[j])
```

Comment:
AFAICT the new code isn't correct here, it gives a very different result. I think you should simply replace `bytestring` with `String`.